### PR TITLE
Model Optimization

### PR DESCRIPTION
**Pull request by Koud-Wind**
## 🤔 What type of PR is this? (check all applicable)

- [x] 🍕 Addition
- [ ] ⌨️ Productivity
- [X] 🐛 Bug Fix
- [X] 🔥 Optimization
- [ ] ⚙️ Configuration
- [x] 🌟 Quality Of Life
- [x] ✨ Enhancement
- [ ] 📝 Documentation

## 📝 Description
Repaired most of the factory files and models for weapons, thanks to Braiam06

The barrel of the M134 and the chamber of the revolver can rotate
Thanks to the M134 model and texture files provided by Braiam06

## 🚦 Testing 
 - [x] Client

## 🖼️ Screenshots/Recordings
Video about rotating parts

https://cdn.discordapp.com/attachments/1128548815994101821/1159447630628257862/Desktop_2023.10.05_-_19.09.39.12.mp4?ex=65310ec0&is=651e99c0&hm=55cc8aa7dfb7045339101bc828756226eb9f7653a8a40f612a062222388fc044&
https://cdn.discordapp.com/attachments/1118470073254944798/1162837063985529013/Desktop_2023.10.15_-_03.38.18.02.mp4?ex=653d6368&is=652aee68&hm=8a25d202e38aac3fbed6b363598d8a8fb12c32448925d4cdef6d1e397e1823a8&

## 📖 Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 Javadoc
- [ ] 🍕 Comments
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by Braiam23** [comment]
bro, don't upload it yet, it needs to be configured

**Comment by Desoroxxx** [comment]
Wat
image

**Comment by Desoroxxx** [comment]
Why is this so big?

**Comment by Braiam23** [comment]
> Why is this so big?

nice question

File: Changelog.md [hunk 1]
```diff
@@ -225,6 +226,27 @@ The new balance pack manager version is `1.3`.
...
+- S7-10 Tricun
+- AS50
+- AUG(WIP)
+
+//  New Guns
+
+- Accuracy International AX50
+- Modernized Kalashnikov Machine Gun(PKM)
+- PGM Hécate II(WIP)
+
```
Comment (Desoroxxx) [inline Changelog.md:249]: No, this either goes in added or changed

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

File: Changelog.md (hunk 1 above)
Comment (Braiam23) [reply Changelog.md:249]: That's all the changes that are added, the reason why I put (WIP) is because there are missing animations

**Comment by Braiam23** [comment]
Nice one

**Comment by Desoroxxx** [comment]
Has been re-created under a cleaner form, new PR: #312

**Pull request closed without merging**